### PR TITLE
Add Spawn Confirmations for Towns and Nations with Spawn Costs

### LINKED ISSUE #3828: Suggestion: Town/nation spawn confirmation for towns/nations with cost for spawn travel
**Please explain your feature request to the best of your abilities:**
Currently towny has a function where mayors/kings can set a cost for their town spawn travel. There is no confirmation nor indication for this until the user has already teleported to the town spawn and had the money taken from them. I propose that a confirmation message appear with the price and require the user to do the command again within some sort of time period (10 seconds?). This would allow for us to actually make use of this feature and not have to worry about scamming.

### PR DESCRIPTION
**Pull request by suneettipirneni**
#### Description: 

This adds confirmations to town nation and outpost spawning *if* there is an associated cost.

Note: This is bypasses the `/t list` and `/n list` clickables as they present the price up front. It also ignores warnings if you are spawning to your own town/nation.

#### New Config Nodes
`global_town_settings.spawn_warnings` - default true, sets whether these types of confirmations should show.
____
This PR includes lang strings.
____
#### Relevant Towny Issue ticket:

Closes #3828

____
- [X] I have tested this pull request for defects on a server. 

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

File: resources/italian.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: Towny
-version: 0.79
+version: 0.78
```
Comment (LlmDl) [inline resources/italian.yml:2]: Version number gone backwards.

File: src/com/palmergames/bukkit/towny/command/NationCommand.java [hunk 2]
```diff
@@ -1365,7 +1372,7 @@ public void mergeNation(Player player, String name) throws TownyException {
 			});
 			
 			// Send confirmation.
-			ConfirmationHandler.sendConfirmation(BukkitTools.getPlayerExact(king.getName()), confirmation);
+			ConfirmationHandler.sendConfirmation(player, confirmation);
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/command/NationCommand.java:1375]: Your branch of confirmations overhaul was missing the commit I made this morning.

**Comment by suneettipirneni** [comment]
@LlmDl were the spawn perms tested?

**Comment by LlmDl** [comment]
@suneettipirneni I didn't do any testing. Is TownSpawnConfirmation.java still needed? I'll test after the changes are made.

File: src/com/palmergames/bukkit/towny/command/NationCommand.java (hunk 2 above)
Comment (suneettipirneni) [reply src/com/palmergames/bukkit/towny/command/NationCommand.java:1375]: Pretty *sure* it's fixed now

Comment (LlmDl) [reply src/com/palmergames/bukkit/towny/command/NationCommand.java:1375]: Still not fixed.

**Pull request merged**